tools/cdn-purge/fastly-purge.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd tools/cdn-purge
+go mod vendor
+FASTLY_KEY=${FASTLY_KEY} go run fastly-purge.go -v
```

Comment:
I don't understand this `FASTLY_KEY=${FASTLY_KEY}` If env var already exists it will be accessible from `fastly-purge.go` , won't it?

Reply:
yes it will, I just want to make it explicit